sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -107,6 +117,14 @@
             .fileSystem(fileSystemName)
             .version(serviceVersion.getVersion())
             .buildClient();
+        String blobUrl = DataLakeImplUtils.endpointToDesiredEndpoint(url, "blob", "dfs");
+        this.blobDataLakeStorageFs = new AzureDataLakeStorageRestAPIImplBuilder()
```

Comment:
If I remember correctly, this is the first Blob API we are calling from the FileSystemClient.